homeassistant/config_entries.py

```diff
@@ -1941,6 +1941,7 @@ def async_create_entry(  # type: ignore[override]
     def async_update_reload_and_abort(
         self,
         entry: ConfigEntry,
+        unique_id: str | None | UndefinedType = UNDEFINED,
```

Comment:
I doubt there is anyone else using this yet, but it would be nice to add the new kwarg at the end

Reply:
Maybe we should make everything after entry kwargs only (add a `*`)